3D&T_ALPHA/3D&T_ALPHA.css

```diff
@@ -5,6 +5,37 @@
     text-align: left;
 }
 
+.charsheet {
+    background-image: url(https://i.imgur.com/5FBZ1bs.jpg);
```

Comment:
Images should be hosted on GitHub and point to the roll 20 character sheet repo.  Please update in a future PR.